src/components/search-page/stores/___helpers__/SearchLocalStore.mockData.ts

```diff
@@ -0,0 +1,31 @@
+export const SearchLocalStoreMockData = [
```

Comment:
@n8rzz where should I put mock data files?

Reply:
For mocks shared by more than one domain, I prefer a *root* dir called `__mock__`.  I'll also put a `__fixtures__` dir at the root as well so that each is easily accessible by other tests that may need them.  For domain-specific mocks, I like another `__mocks__` dir near the domain (like `stores`, etc).  

There very likely are cases where root a `__mocks__` dir doesn't make sense, but frequently you will have an entity used in multiple domains across an application.  Think of a `Project` model that would be needed for tests within the `stores`, but also tests with components (`src/stores` and `src/components`).